src/query/xasl_to_stream.c

```diff
@@ -5985,8 +5973,7 @@ xts_sizeof_xasl_node (const XASL_NODE * xasl)
       return ER_FAILED;
     }
 
-  size += (OR_INT_SIZE		/* projected_size */
-	   + OR_DOUBLE_ALIGNED_SIZE	/* cardinality */
+  size += (OR_DOUBLE_ALIGNED_SIZE	/* cardinality */
```

Comment:
Is there a reason why xasl->cardinality is excluded but included in the size calculation?

Reply:
thank you. I've omitted